subsys/bluetooth/host/mesh/cfg_srv.c

```diff
@@ -1122,12 +1108,58 @@ static u8_t va_add(u8_t *label_uuid, u16_t *addr)
 		return STATUS_UNSPECIFIED;
 	}
 
+	free_slot->ref = 1;
 	free_slot->addr = *addr;
 	memcpy(free_slot->uuid, label_uuid, 16);
 
 	return STATUS_SUCCESS;
 }
 
+static u8_t va_del(u8_t *label_uuid, u16_t *addr)
+{
+	int i;
+
+	if (addr) {
+		*addr = BT_MESH_ADDR_UNASSIGNED;
```

Comment:
Minor nitpick: do this after the for-loop so you avoid writing to the variable twice when a match is found.